cucumber/authenticators_config/features/authn_config.feature

```diff
@@ -125,3 +125,106 @@ Feature: Authenticator configuration
     """
     Errors::Authentication::AuthenticatorNotSupported
     """
+
+  Scenario: Authenticator without service-id is successfully configured
+    When I am the super-user
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-gcp
+      body:
+      - !webservice
+    """
+    And I save my place in the log file
+    And I PATCH "/authn-gcp/cucumber" with body:
+    """
+    enabled=true
+    """
+    Then the HTTP response status code is 204
+    And authenticator "cucumber:webservice:conjur/authn-gcp" is enabled
+
+    When I successfully PATCH "/authn-gcp/cucumber" with body:
+    """
+    enabled=false
+    """
+    Then the HTTP response status code is 204
+    And authenticator "cucumber:webservice:conjur/authn-gcp" is disabled
+
+  Scenario: Authenticator without service-id and webservice does not exist
+    When I am the super-user
+    And I save my place in the log file
+    And I PATCH "/authn-gcp/cucumber" with body:
+    """
+    enabled=true
+    """
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    Errors::Authentication::Security::WebserviceNotFound
+    """
+    And authenticator "cucumber:webservice:conjur/authn-gcp" is disabled
+
+    When I load a policy:
+    """
+    - !policy
+      id: conjur/authn-gcp
+    """
+    And I save my place in the log file
+    And I PATCH "/authn-gcp/cucumber" with body:
+    """
+    enabled=true
+    """
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    Errors::Authentication::Security::WebserviceNotFound
+    """
+    And authenticator "cucumber:webservice:conjur/authn-gcp" is disabled
+
+  Scenario: Authenticated user can not update authenticator without service-id
```

Comment:
the headline is misleading , did you meant to test here an unauthorized user ?

Reply:
yes fixed the scenario name